library/src/main/java/com/novoda/downloadmanager/lib/DownloadClientReadyChecker.java

```diff
@@ -5,9 +5,10 @@
 
     /**
      * This method is executed on a background thread
-     * @param collatedDownloadInfo
+     *
+     * @param downloadBatch
```

Comment:
don't add jdoc params unless you are going to fill them in with useful information